### DISCUSSION TITLE
How long will my domain on pending verification after adding the TXT record?

### DISCUSSION THREAD

**Discussion by aitorMau** [vercel/community#3242] [Help] [answered]
### Summary

Problems with domain verification after adding the TXT records

### Steps to Reproduce

Hi I'm trying to verify a domain and it doesn't change the state..it is constantly saying "Domain is pending verification".

![B3korUdvp0](https://github.com/vercel/community/assets/2556607/74218502-a5f6-49c1-8857-91fc0d115455)

I've already added the TXT records

![chrome_9BF7N6Bbbl](https://github.com/vercel/community/assets/2556607/e39b6c5c-9617-40d4-89bf-77ab043b3d39)

Am I doing something wrong? Thanks

**Comment by swarnava** [comment, marked as answer]
Hi,

It should be available as soon as the [DNS propagated](https://vercel.com/docs/concepts/projects/domains/working-with-dns#dns-propagation) expected value: https://dnschecker.org/#TXT/_vercel.pentridgeprisontours.com.au. I can see that it hasn't propagated yet.

**Reply by themightywonton** [reply]
hi! it is been more than 3 days...why is not propagated already? do you know?
I'm pretty sure there is something wrong.
Thanks

**Comment by aitorMau** [comment]
hi! it is been more than 3 days...why is not propagated already? do you know?
I'm pretty sure there is something wrong. 
Anyone can help?
Thanks

**Reply by swarnava** [reply]
You need to reach out to the Support team from where you purchased the domain. Since vercel doesn't manage your Nameserver or DNS, we don't have much context why it's taking so long.

**Reply by ee-blog** [reply]
https://dnschecker.org/#TXT/_vercel.eebk.com

Please help me, I am encountering the same problem.

**Reply by swarnava** [reply]
I don't see this domain added to any project yet https://vercel.com/docs/concepts/projects/domains/add-a-domain

**Reply by ee-blog** [reply]
![image](https://github.com/vercel/community/assets/52785845/ddb48de1-50b6-4f5f-b0ca-8a0013051d19)

**Reply by swarnava** [reply]
Thanks, the TXT value is incorrect, there is a missing `,` which is unexpected

**Reply by ee-blog** [reply]
I think it's an issue with the DNS service provider. If I add a comma, it will be deleted. Do you know of any other solution to this problem?

**Reply by ee-blog** [reply]
Hello, I have switched my DNS service provider and I'm now facing a new issue. Whenever I enable the CDN service on my DNS server, the system identifies an abnormality with the CNAME record for my custom domain.
